src/Datatable/Datatable.php

```diff
@@ -484,10 +486,10 @@ public function setGetDataUrl($defaultUrl = null)
             };
             GET_DATA;
         } else {
-            // @todo setConfig type POST
             $this->getDataTemplate = <<<GET_DATA
                 let getData = async () => {
                     return {
+                        headers: { 'X-CSRF-Token': '{$csrfToken}' },
```

Comment:
shall we check if csrfToken is not null before adding it to the headers?

Reply:
added checking that is not null csrfToken to add in headers